scripts/mark-the-letters.js

```diff
@@ -94,7 +94,7 @@ H5P.MarkTheLetters = (function($, Question, UI) {
             self.setFeedback(feedback, score, max, self.params.scoreBarLabel);
             self.$progressBar = UI.createScoreBar(max, 'scoreBarLabel');
             self.$progressBar.setScore(score);
-            self.$progressBar.appendTo($feedbackDialog);
+            self.$progressBar.appendTo($feedbackContainer);
```

Comment:
Should be careful while copying the code. Feedback dialog is contains some other elements and hidden sometimes.